Examples/AppDelegate.m

```diff
@@ -20,6 +21,13 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
         self.window.layer.speed = 100;
     }
 
+    NSUInteger maxCacheSize = 62914560;
+    [[MGLOfflineStorage sharedOfflineStorage] setMaximumAmbientCacheSize:maxCacheSize withCompletionHandler:^(NSError * _Nullable error) {
```

Comment:
Hrm, can you explain why we’re doing this globally in the app delegate?

Reply:
I put that in AppDelegate initially in order to have it happen before any map view is loaded, but I think I can move it to the actual example.